### PR TITLE
Escape global search query, refs #12995

### PR DESCRIPTION
**Pull request by mcantelon**
Added entity escaping when showing, as a form input value, the search query.

### CODE REVIEW

File: apps/qubit/modules/search/templates/_box.php [hunk 1]
```diff
@@ -8,9 +8,9 @@
     <input type="hidden" name="sort" value="relevance"/>
 
     <?php if (isset($repository) && !sfConfig::get('app_enable_institutional_scoping')): ?>
-      <input type="text" name="query"<?php if (isset($sf_request->query)) echo ' class="focused"' ?> value="<?php echo $sf_request->query ?>" placeholder="<?php echo __('Search %1%', array('%1%' => strip_markdown($repository))) ?>"/>
```
Comment (sbreker) [inline apps/qubit/modules/search/templates/_box.php:11]: Looks good!  I think this change needs to also be made in two additional places:
- In the equivalent file in the Archives Canada plugin
- ~line 30 in: apps/qubit/modules/repository/templates/_holdingsInstitution.php

**Review by sbreker — CHANGES_REQUESTED** [review]

File: apps/qubit/modules/search/templates/_box.php (hunk 1 above)
Comment (mcantelon) [reply apps/qubit/modules/search/templates/_box.php:11]: Thanks Steve! Will do.

Comment (mcantelon) [reply apps/qubit/modules/search/templates/_box.php:11]: (I've committed escaping to those other templates.)

**Comment by mcantelon** [comment]
@sbreker I've made those changes... thanks!

**Review by sbreker — APPROVED** [review]
Looks good @mcantelon - thanks!

**Comment by mcantelon** [comment]
Thanks @sbreker !

**Pull request merged**